dev/com.ibm.ws.webcontainer.servlet.4.0_fat/bnd.bnd

```diff
@@ -43,20 +40,19 @@ src: \
 fat.project: true
 
 tested.features:\
-    servlet-5.0,\
```

Comment:
good catch looks like a mistake, running locally to validate.

Reply:
I think some of this might be able to be cleaned up even more we only need to list features here that are not included in the server.xmls directly. Let me run and see what is necessary and push up a change.

Reply:
I'm going to remove the tested.features from each of the buckets and run them and see what the test output is. I think I was able to remove the Servlet 5.0 and Pages 3.1 features b/c they exist already in one of the server.xmls within the bucket.

Reply:
https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet50_GetContextPath/server.xml